src/App.vue

```diff
@@ -21,6 +21,16 @@ export default {
   components: {
     TheNav,
     TheFooter
+  },
+  data() {
+    return {
+      isBodyLocked: false
+    }
+  },
+  methods: {
+    handleBody(isLocked) {
+      this.isBodyLocked = isLocked
+    }
```

Comment:
This change is unintended, it should be removed from this PR (you can literally create a new commit where it's deleted.

Reply:
**Done**

I will deal with the rest of your suggestions tomorrow since the addition of the error toast will probably require some more time than a few minutes but I did this one today so I don't forget tomorrow.